Src/PChecker/CheckerCore/CheckerMode.cs

```diff
@@ -16,5 +16,9 @@ public enum CheckerMode
     /// <summary>
     /// Mode for exhaustive explicit-state search with state-space coverage reporting
     /// </summary>
-    Coverage
+    Coverage,
```

Comment:
Can we drop coverage?

Reply:
Let's drop this once we have a minimal running version. I wish to retain this for experimental comparison purposes. Note that this is hidden from P CLI.